benchmark_runner/common/prometheus/metrics-default.yaml

```diff
@@ -1,267 +1,10 @@
-# API server
-- query: histogram_quantile(0.99, sum(rate(apiserver_request_duration_seconds_bucket{apiserver="kube-apiserver", verb!~"WATCH", subresource!="log"}[2m])) by (verb,resource,subresource,instance,le)) > 0
-  metricName: API99thLatency
 
-- query: sum(irate(apiserver_request_total{apiserver="kube-apiserver",verb!="WATCH",subresource!="log"}[2m])) by (verb,instance,resource,code) > 0
-  metricName: APIRequestRate
+# container CPU/Memory
 
-- query: sum(apiserver_current_inflight_requests{}) by (request_kind) > 0
-  metricName: APIInflightRequests
```

Comment:
This may be of interest for boot storm testing, where there will be a lot of objects created simultaneously.

Reply:
I run just 2 queries CPU and memory and we need more queries I will add it later